content/blog/2017/2017-02-14-pipleine-editor-preview.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Say 'Hello' to the Blue Ocean Pipeline Editor"
+tags:
+- blueocean
+- editor
+- declarative
+- pipeline
+author: jamesdumay
+---
+
+link:/blog/2016/09/19/blueocean-beta-declarative-pipeline-pipeline-editor/[Back in September 2016 we announced the availability of the Blue Ocean beta]
+and the forthcoming Pipeline Editor. We are happy to announce that you can try
+the Pipeline Editor preview today.
+
+=== What is it?
+
+The Pipeline Editor is the simplest way for anyone wanting to get started with
+creating Pipelines in Jenkins. It's also great way for advanced Jenkins users
+to start adopting Pipeline. It allows developers to break up their Pipeline into different
+ stages and parallelize tasks that can occur at the same time - graphically.
+ The rest is up to you.
+
+Thats right, the days of writing Pipelines in your text editor are over. You can do it
+all from Blue Ocean and have it produce a link://blog/2017/02/03/declarative-pipeline-ga/[Declarative Pipeline]
+ `Jenkinsfile` for you. Not sure what a Jenkins Pipeline is? Why not checkout our link:/doc/[new guided tour]!
+
+image::/images/blueocean/editor-1.png["The Editor", role="center"]
+
+=== Whats next?
+
+`Jenkinsfile`'s belong in the Git repository so that it is versioned a long side
```

Comment:
hmm maybe the github preview is just bad here, assume you looked at the asciidoc version and it's fine.